sips/validators_per_operator_1k_cap.md

```diff
@@ -0,0 +1,61 @@
+|     Author     |               Title               | Category  |       Status        |    Date    |
+| -------------- | --------------------------------- | --------- | ------------------- | ---------- |
+| Matheus Franco | 1K cap on validators per operator | Contracts | open-for-discussion | 2024-11-28 |
+
+## Summary
+
+This SIP proposes increasing the cap on the number of validators per operator from 500 to 1000.
+
+## Motivation
+
+The recent optimizations allowed operators to reduce their operational costs while running the same number of validators, thus keeping the same profits. Therefore, this allows operators to further increase the size of their validator sets, keeping a sustainable processing cost and increasing their profitability.
+
+## Performance trade-offs
+
+To understand the performance impact, we compared two settings with the same number of validators (namely, 104k), one with a cap of 500 and the other of 1000 validators per operator. Using the 500 case as a benchmark, we got the following results for the 1000 case:
+
+| Metric                             | Impact     |
+|------------------------------------|------------|
+| Maximum operator cryptography cost | 20% higher |
+| Average operator cryptography cost | 80% higher |
```

Comment:
I'll add the following explanation in the text:
- if we didn't have Committee Consensus, the average would be almost 100% higher (note that non-committee cryptography has less relevance here). But committee consensus optimizes this to 80% on average.
- Why not better than 80%? When you have few validators, committee consensus doesn't have much impact. So, for the average operator, it's not that significant.
- The maximum case is the case of an operator that had 500 validators in the worst topic, and now it has 1000. The effects of committee consensus are boosted because the number of consensus duties approaches a fixed number of 32 per epoch, even though you doubled the number of validators.
Let me know if it makes sense for you? Btw, these are the numbers from the simulations. Above is my interpretation of it, not the ground truth.